background_scripts/completion.coffee

```diff
@@ -23,21 +23,28 @@ class Suggestion
     @title ||= ""
     # When @autoSelect is truthy, the suggestion is automatically pre-selected in the vomnibar.
     @autoSelect = false
+    # If @noHighlightTerms is falsy, then we don't highlight matched terms in the title and URL.
+    @noHighlightTerms = false
+    # If @insertText is a string, then the indicated text is inserted into the vomnibar input when the
+    # suggestion is selected.
+    @insertText = null
 
   computeRelevancy: -> @relevancy = @computeRelevancyFunction(this)
 
   generateHtml: ->
     return @html if @html
     relevancyHtml = if @showRelevancy then "<span class='relevancy'>#{@computeRelevancy()}</span>" else ""
+    highlightTerms =
```

Comment:
Can this be renamed to `escapeAndHighlight` or similar, so it's obvious from the point of call that there aren't XSS issues?